### PR TITLE
ISSUE #4661 - Use A-Site privacy page

### LINKED ISSUE #4661: Use Asite's privacy policy instead of having our own
### Description
Tiago has requested us to replace our existing privacy policy with Asite's. Instead of making a copy, we should just redirect our users to the Asite's page so we don't have to do this again!

Our current policy: https://www.3drepo.io/v5/privacy 
Asite's: https://www.asite.com/privacy-policy

### Goals
- [ ] As the CISO of Asite, I want our privacy policy to be aligned across all products
- [ ] As a user of 3D Repo, I want to view the latest, most accurate privacy policy

### Tasks
TBD

### PR DESCRIPTION
**Pull request by Amantini1997**
This fixes #4661 

#### Description
Use Asite privacy page instead of 3dRepo
Trying to access privacy page will result in a redirect to ASite privacy page

#### Test cases
Try to access the privacy page via different links:
- copy paste link in the URL bar;
- footer;
- navlink from cookies/terms page

### CODE REVIEW

**Review by The-Daniel — DISMISSED** [review]

File: frontend/src/v5/ui/components/shared/sso/microsoftText.component.tsx [hunk 1]
```diff
@@ -59,7 +59,7 @@ export const MicrosoftText = ({ title, className }: MicrosoftTextProps) => (
 				`}
 				values={{
 					TermsLink: (label) => <Link to={TERMS_ROUTE} target="_blank">{label}</Link>,
-					PrivacyLink: (label) => <Link to={PRIVACY_ROUTE} target="_blank">{label}</Link>,
+					PrivacyLink: (label) => <a href={PRIVACY_ROUTE} target="_blank">{label}</a>,
```
Comment (carmenfan) [inline frontend/src/v5/ui/components/shared/sso/microsoftText.component.tsx:62]: The link is rendered differently, can we unify it?

on staging
image
on this branch
image

File: frontend/src/v5/ui/components/legal/LegalLayout/legalAppBar/legalAppBar.component.tsx [hunk 2]
```diff
@@ -35,9 +35,9 @@ export const LegalAppBar = ({ activePage }: ILegalAppBar): JSX.Element => (
 				<LogoIcon />
 			</Link>
 			<NavLinks>
-				<NavLink to="/v5/privacy" selected={activePage === 'privacy'}>
+				<PrivacyLink href={PRIVACY_ROUTE} target="_blank">
```
Comment (carmenfan) [inline frontend/src/v5/ui/components/legal/LegalLayout/legalAppBar/legalAppBar.component.tsx:38]: Can we fix the formatting here too:
staging:
image

this branch:
image

**Review by carmenfan — CHANGES_REQUESTED** [review]

**Review by carmenfan — CHANGES_REQUESTED** [review]
the footer is not going to the privacy page

**Review by carmenfan — APPROVED** [review]

**Review by The-Daniel — APPROVED** [review]

**Pull request merged**